.github/workflows/build_tiny_todo_reusable.yml

```diff
@@ -0,0 +1,46 @@
+name: Build and Test TinyTodo
+on:
+    workflow_call:
+      inputs:
+        cedar_policy_ref:
+          required: false
+          type: string
+
+jobs:
+  build_and_test_tiny_todo:
+    name: Build and Test TinyTodo
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        toolchain:
+          - stable
+    steps:
+      - name: Checkout Cedar Examples
+        uses: actions/checkout@v3
+      - name: Checkout cedar-policy
+        uses: actions/checkout@v3
+        with:
+          repository: cedar-policy/cedar
+          ref: ${{ inputs.cedar_policy_ref }}
+          path: ./cedar
+# If we passed a branch for cedar-policy, modify Cargo.toml to use that branch
+      - name: Replace Crates dot IO with Github version
+        if: "${{ inputs.cedar_policy_ref != '' }}"
+        run: cd tinytodo && printf "\npath = \"../cedar/cedar-policy\"" >> Cargo.toml
```

Comment:
I did some searching for a better way top do this update. It looks like editing `Cargo.toml` is the only option, but we could instead edit it by appending [a `[patch]` section](https://doc.rust-lang.org/cargo/reference/overriding-dependencies.html#the-patch-section). This would be a little less brittle to future changes to `Cargo.toml` since it wouldn't depend on `[dependencies.cedar-policy]` being the last section in the file.

Reply:
reading your link (documenting the `[patch]` section), it says

> The `[patch]` table can also be specified as a [configuration option](https://doc.rust-lang.org/cargo/reference/config.html), such as in a `.cargo/config.toml` file or a CLI option like `--config 'patch.crates-io.rand.path="rand"'`. This can be useful for local-only changes that you don’t want to commit, or temporarily testing a patch.

so that would be a solution that doesn't require editing Cargo.toml

Reply:
I just went with John's suggestion so we don't have to pass that `--config` to each cargo command.